addons/rating/views/rating_rating_views.xml

```diff
@@ -228,10 +228,10 @@
                 <search string="Ratings">
                     <field name="rated_partner_id"/>
                     <field name="rating"/>
+                    <field name="parent_res_name" filter_domain="[('parent_res_name', 'ilike', self)]"/>
                     <field name="partner_id"/>
                     <field name="res_name" filter_domain="[('res_name','ilike',self)]"/>
                     <field name="res_id"/>
-                    <field name="parent_res_name" filter_domain="[('parent_res_name','ilike',self)]"/>
```

Comment:
In fact, why did you do that changes? 🤔 
It does not seem to be in the spec of the task related, right? 🧐

Reply:
> In fact, why did you do that changes? 🤔 It does not seem to be in the spec of the task related, right? 🧐

This change was related to the search view field sequence so I moved these changes to the helpdesk.
thanks